openedx/core/djangoapps/user_api/views.py

```diff
@@ -44,7 +47,7 @@ class LoginSessionView(APIView):
     # so do not require authentication.
     authentication_classes = []
 
-    @method_decorator(ensure_csrf_cookie)
+    @method_decorator(ensure_csrf_cookie_cross_domain)
```

Comment:
What does cross_domain actually allow?  Is the fact that sandboxes and other test environments share the same domain as Production a problem for this solution?